website/docs/interactors/8-storybook.md

```diff
@@ -0,0 +1,38 @@
+---
+id: storybook
+title: Storybook
+---
+
+Interactors not only make writing tests easier, it can also help you develop UI components. With the upcoming release of [`Component Story Format 3.0`](https://storybook.js.org/blog/component-story-format-3-0/), you will be able to use Interactors in [`Storybook`](https://storybook.js.org/).
+
+This requires no additional setup. Just install `@bigtest/interactor` to your project, and then you can use interactors in your stories immediately.
+
+In the same way Interactors make tests more reliable and easier to read, it will also enhance your developer experience with Storybook.
+
+Here is an example of how a story can be written with interactors:
+```js
+import { Button, TextField } from '@bigtest/interactor';
+
+export const FormSignIn = {
+  play: async () => {
+    await TextField('Email').fillIn('homer@gmail.com');
+    await TextField('Password').fillIn('donuts123');
+    await Button('Sign In').click();
+  }
+};
+```
+
+And this is how you would normally write that story:
+```js
+import { screen } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+
+export const FormSignIn = {
+  play: () => {
+    userEvent.type(screen.getById('email'), 'homer@gmail.com');
+    userEvent.type(screen.getById('password'), 'donuts123');
+    userEvent.click(screen.getByText('Sign In'));
+  }
+};
+```
```

Comment:
I suggest swapping examples. It's better to say, how Storybook suggests writing `play` methods and how they might look better with interactions.